### ISSUE TITLE
ARM Support?

### ISSUE THREAD

**Issue by BillyDM** [Lokathor/safe_arch#98]
Is support for ARM intrinsics planned?

**Comment by Lokathor** [comment]
I would like to have them, though there are very many and so I doubt that I'll get through much of it on my own.

The main thing is that they're not yet stable, and I would like to keep the base crate working on stable. If you PR them yourself, please put modules with ARM intrinsics behind a feature flag for now.

**Comment by Lokathor** [comment]
I was just informed that neon for aarch64 is stable now (and has been a while), there's just been a docs bug so it looks unstable in the docs.

Accordingly, I'll try to start adding those functions "soon". My physical access to an aarch64 device is more sporadic than with my normal computer so I'm not sure how fast progress will go.

**Comment by CeleritasCelery** [comment]
Would you accept a PR that adds ARM intrinsics support?

**Comment by Lokathor** [comment]
Yeah, definitely. Remember to add a CI run that at least builds the ARM version. Hopefully tests can be run in some way as well, but at least a build pass is essential.
